### PR TITLE
Feature phone number format

### PR DESCRIPTION
**Pull request by mrave001**
Hey everyone, can someone review this PR for me?

### CODE REVIEW

**Comment by bclincy** [comment]
You going to have to add .vscode to the gitignore and uncommit the files like .DS

File: docker/etc/nginx/default.conf [hunk 1]
```diff
@@ -3,7 +3,7 @@
 server {
     listen 80 default_server;
```
Comment (bclincy) [inline docker/etc/nginx/default.conf:4]: Uncommitted these changes

File: docker/web/app/src/Foo.php [hunk 2]
```diff
@@ -22,16 +22,16 @@ public function downloadImg(string $url, string $name, string $path = __DIR__ .
         if (file_exists($saveTo) === true){
             unlink($saveTo);
         }
-        $fp = fopen($saveTo,'w+');
-        fwrite($fp, $raw);
-        fclose($fp);
+        // $fp = fopen($saveTo,'w+');
+        // fwrite($fp, $raw);
```
Comment (bclincy) [inline docker/web/app/src/Foo.php:26]: These commented out?

File: docker/web/public/finalproject/index.php [hunk 3]
```diff
@@ -47,6 +48,11 @@ function stateSelector (array $states) {
    if ($photo !== false) {
       $vcard->addPhoto($uploadImgPath . $photo);
    }
+
+   // format phone number
+   $mobile = $foo->format_phone_us($mobile);
+   // var_dump($phone);
```
Comment (bclincy) [inline docker/web/public/finalproject/index.php]: You can remove the debugger code

**Review by bclincy — CHANGES_REQUESTED** [review]

File: docker/web/app/src/Foo.php (hunk 2 above)
Comment (mrave001) [reply docker/web/app/src/Foo.php:26]: yeah while I was trying to diagnose why the code wasn't working you asked me to comment those. Should I un comment?

File: docker/web/public/finalproject/index.php (hunk 3 above)
Comment (mrave001) [reply docker/web/public/finalproject/index.php]: Ok.

File: docker/web/app/src/Foo.php (hunk 2 above)
Comment (mrave001) [reply docker/web/app/src/Foo.php:26]: Yeah this was commented out while I was trying to diagnose why the code wasn't working and you suggested commenting these out.

File: docker/web/public/finalproject/index.php (hunk 3 above)
Comment (mrave001) [reply docker/web/public/finalproject/index.php:54]: Ok.

**Review by bclincy — APPROVED** [review]

**Review by bclincy — APPROVED** [review]

**Pull request merged**